docs/source/guide/guide-error-mitigation.rst

```diff
@@ -80,10 +80,35 @@ applicable thus depends on the performance of the underlying hardware as well as
 of when zero-noise extrapolation is effective, and how effective it is, is the subject of ongoing research.
 
 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Probabilistic error cancellation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Probabilistic error cancellation uses the quasi--probability representation :cite:`Temme_2017_PRL` to write an ideal quantum 
+channel as a linear combination of noisy operations. Given a set of noisy but implementable operations :math:`\Omega = \{O_1, \dots, O_m\}`, an ideal quantum 
+channel can be written as :math:`\mathcal{U}_i = \sum_{\alpha} P_i(\alpha)\sigma_i(\alpha) \mathcal{O}_\alpha`, where :math:`P_i(\alpha)` is a probability 
+distribution, and the coefficients take values :math:`\sigma_i(\alpha)=\pm 1`. Using this method we can estimate the ideal expectation value of some observable of 
+interest :math:`\langle X\rangle_{\text{ideal}}`, under the action of circuit given by a sequence of ideal quantum gates :math:`\{\mathcal{U}_i\}_{i}^{L}`, by 
+sampling for each each quantum gate :math:`\mathcal{U}_i` a noisy operation :math:`\mathcal{O}_{\alpha_i}` with probability 
+:math:`|P_i(\alpha)\sigma_i(\alpha)|/\sum_{\alpha}P_i(\alpha)\sigma_i(\alpha)|`. Then given a final state of the non--ideal circuit :math:`\rho_f`, measuring the 
+observable :math:`X`, and setting :math:`\gamma_i := \sum_{\alpha} P_i(\alpha)\sigma_i(\alpha)|`, :math:`\gamma_{\text{tot}} := \sum_{i} \gamma_i|` and 
+:math:`\sigma_{\text{tot}} = \prod_{i=1}^L \sigma_i(\alpha)` we obtain an unbiased estimator of the ideal expectation value as :math:`\langle 
+X\rangle_{\text{ideal}} = \langle \gamma_{\text{tot}} \sigma_{\text{tot}} \mu(X) \rangle_{\text{sample}}`, where :math:`\mu(X)` is a random variable for the 
+measurement outcome and :math:`\langle \cdot\rangle_{\text{sample}}` is the expectation value of the sample average of the procedure described. This procedure 
+effectivelly cancels the effects of the noise process by probabalistically applying its inverse.
+
+The samples can be generated by drawing according to the probability distribution :math:`P_i(\alpha)` using Monte Carlo methods. The number of samples required to 
+estimate the ideal expectation value with error :math:`\delta` and probability :math:`1-\epsilon` is :cite:`Takagi2020optimal` :math:`\left(2 
+\gamma_{\text{tot}}^{2} / \delta^{2}\right) \log (2 / \epsilon)`. Thus, the sampling overhead is determined by :math:`\gamma_{\text{tot}}` which grows 
+exponentially in the number of gates. It is then crucial to find a linear decomposition that minimizes :math:`\gamma_{\text{tot}}`. In addition, a full 
+characterization of the noisy operations up to a good precision is required, which can be costly depending on the implementation. Optimal decomposition in the 
+sepcial case of depolarizing noise is detailed in :cite:`Temme_2017_PRL`. The methods are contained in the :mod:`mitiq.pec` and :mod:`mitiq.pec.depolarization` 
+modules.
+ 
```

Comment:
```suggestion
In Mitiq, this technique is implemented in the module :mod:`mitiq.zne`.

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Probabilistic error cancellation
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Probabilistic error cancellation uses a quasi-probability representation :cite:`Temme_2017_PRL` to express an ideal quantum 
channel as a linear combination of noisy operations. Given a set of noisy but implementable operations :math:`\Omega = \{O_1, \dots, O_m\}`, an ideal unitary gate 
can be expressed as :math:`\mathcal{G} = \sum_{\alpha} \eta_{\alpha} \mathcal{O}_\alpha = \gamma \sum_{\alpha} P(\alpha) \sigma(\alpha) \mathcal{O}_\alpha`, where
:math:`\eta_\alpha` are real coefficients, :math:`\gamma = \sum_{\alpha} |\eta_\alpha|`, :math:`P(\alpha)=|\eta_\alpha | /\gamma` is a probability 
distribution, and :math:`\sigma(\alpha)={\rm sign}(\eta_\alpha)`.

In this setting, we would like to estimate the ideal expectation value of some observable of interest :math:`\langle X\rangle_{\text{ideal}}`, 
after the action of an ideal circuit given by a sequence of ideal quantum gates :math:`\{\mathcal{\mathcal G}_i\}_{i=1}^L`. This can be achieved by 
sampling for each ideal gate :math:`\mathcal{G}_i` a noisy operation :math:`\mathcal{O}_{\alpha}` with probability 
:math:`P_i(\alpha)`. This random sampling will produce a noisy circuit (given by the sequence of sampled operations :math:`\{\mathcal{O}_{\alpha_i}\}_{i=1}^L`)
whose execution produces the final mixed state :math:`\rho_f`.
Then, by measuring the observable :math:`X`, setting :math:`\gamma_{\text{tot}} := \prod_{i}^L \gamma_i` and 
:math:`\sigma_{\text{tot}} = \prod_{i=1}^L \sigma_i(\alpha)`, one can obtain an unbiased estimate of the ideal expectation value as :math:`\langle 
X\rangle_{\text{ideal}} =  \mathbb E \left[ \gamma_{\text{tot}} \sigma_{\text{tot}} X_{\rm noisy} \right]`, where :math:`X_{\rm noisy}` is
the experimental estimate of :math:`{\rm tr}[\rho_f X]` and :math:`\mathbb E` is the sample average over many repetitions of the previous procedure.

In Mitiq, this technique is implemented in the module :mod:`mitiq.pec`.

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Limitations of probabilistic error cancellation
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

The number of samples required to estimate the ideal expectation value with error :math:`\delta` and probability :math:`1-\epsilon` scales as 
:math:`\left(2 \gamma_{\text{tot}}^{2} / \delta^{2}\right) \log (2 / \epsilon)`  :cite:`Takagi2020optimal`. Thus, the sampling overhead is determined 
by :math:`\gamma_{\text{tot}}` which grows exponentially in the number of gates. It is then crucial to find a linear decomposition that minimizes :math:`\gamma_{\text{tot}}`. 
In addition, a full characterization of the noisy operations up to a good precision is required, which can be costly depending on the implementation.
```

Reply:
I apologize for suggesting the full block instead of individual lines but this solution was much more practical.

Reply:
Thanks, I agree with the suggestions and have committed them.